src/mats-method-workspace/mats-method-checks.service.spec.ts

```diff
@@ -69,7 +69,7 @@ describe('Mats Method Checks Service Test', () => {
 
       payload.beginDate = new Date('2023-02-28');
       payload.beginHour = 5;
-      payload.endDate = moment(getMaximumFutureDate()).add(1, 'days');
+      payload.endDate = moment(getMaximumFutureDate).add(1, 'days');
```

Comment:
This _should_ call the function:
```suggestion
      payload.endDate = moment(getMaximumFutureDate()).add(1, 'days');
```

Reply:
just noticed that; made the correction and updated accordingly. I used a script to update that was the issue.

Reply:
Ah okay, that makes sense.